app/konnect/mesh-manager/service-mesh.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Create a service mesh
+content_type: how-to
+beta: true
+---
+
+Creating a fully-functioning {{site.mesh_product_name}} deployment in {{site.konnect_short_name}} involves the following steps:
```

Comment:
What's the rationale of this doc? looks to 99% the same as: app/konnect/getting-started/configure-mesh.md

Reply:
@lahabana This doc's purpose is just to provide a general "how to" like we do for other features in Konnect while the getting started Mesh doc is more opinionated and geared for getting people up and running quickly with mesh.